java-allocation-instrumenter/pom.xml

```diff
@@ -223,4 +223,36 @@
       <organization>Google Inc.</organization>
     </developer>
   </developers>
+  
+  <profiles>
+    <profile>
+      <id>jdk8</id>
+      <activation>
+        <jdk>[1.8,)</jdk>
+      </activation>
+      <!-- Disable doclint under JDK 8 -->
```

Comment:
Okay.  That seems like the kind of thing we should fix, but we can fix it later.